File: beginner/__2000__/2747/2747.cpp
#include <iostream>

using namespace std;

int main(){

    string traces, traces2 = "|";

    for(int i = 0; i < 39; i++){
        traces += "-";
    }

    
    for(int i = 0; i < 37; i++){
        traces2 += " ";
    }

    traces2 += "|";

    cout << traces << endl;

    cout << traces2 << endl;
    cout << traces2 << endl;
    cout << traces2 << endl;
    cout << traces2 << endl;
    cout << traces2 << endl;

    cout << traces << endl;

    return 0;
}

File: beginner/__2000__/2747/2747.java
import java.io.IOException;


public class Main {
	public static void main(String[] args) throws IOException {

		String traces = "";
        String traces2 = "|";

        for(int i = 0; i < 39; i++){
            traces += "-";
        }

        
        for(int i = 0; i < 37; i++){
            traces2 += " ";
        }

        traces2 += "|";

        System.out.println(traces);

        System.out.println(traces2);
        System.out.println(traces2);
        System.out.println(traces2);
        System.out.println(traces2);
        System.out.println(traces2);

        System.out.println(traces);
	}
}